database.sql

```diff
@@ -959,13 +959,16 @@ CREATE TABLE IF NOT EXISTS `openwebauth-token` (
 -- TABLE parsed_url
 --
 CREATE TABLE IF NOT EXISTS `parsed_url` (
-	`url` varbinary(255) NOT NULL COMMENT 'page url',
+    `url-hash` char(64) NOT NULL COMMENT 'page url hash',
```

Comment:
The `database.sql` doesn't fit to the `dbstructure.config.php`. It would be better to create it via the command line command `bin/console dbstructure dumpsql > database.sql`

Reply:
I will never do that correctly.